.styleci.yml

```diff
@@ -12,3 +12,7 @@ disabled:
   - psr0
   - unalign_double_arrow
   - unalign_equals
+
+finder:
+  not-name:
+    - "*.php.twig"
```

Comment:
That was I thinking. But no, php-cs-fixer with default config was trying to fix my `.php.twig`...